build/package/yum.repos.d/adoptopenjdk.repo

```diff
@@ -0,0 +1,6 @@
+[AdoptOpenJDK]
```

Comment:
I think we can delete this file now, right?

Reply:
ah yes, not required anymore